modules/statistics/ajax/GetCSV.php

```diff
@@ -15,61 +26,69 @@
 header("Content-type: text/plain");
 $DB = Database::singleton();
 $ConditionBindings = array();
-if(isset($_REQUEST['InstrumentY'])) {
+if (isset($_REQUEST['InstrumentY'])) {
     $Instrument = $_REQUEST['InstrumentY'];
 }
-if(isset($_REQUEST['InstrumentX'])) {
+if (isset($_REQUEST['InstrumentX'])) {
     $InstrumentX = $_REQUEST['InstrumentX'];
 }
-if(isset($_REQUEST['FieldY']) && !empty($_REQUEST['FieldY'])) {
+if (isset($_REQUEST['FieldY']) && !empty($_REQUEST['FieldY'])) {
     $Field = "i." . $_REQUEST['FieldY'];
 } else {
     $Field = "Candidate_Age";
 }
-if(isset($_REQUEST['FieldX']) && !empty($_REQUEST['FieldX'])) {
+if (isset($_REQUEST['FieldX']) && !empty($_REQUEST['FieldX'])) {
     $FieldX = $_REQUEST['FieldX'];
 } else {
     $FieldX = 'Candidate_Age';
 }
-if(isset($InstrumentX)) {
+if (isset($InstrumentX)) {
     $FieldX = "i2." . $FieldX;
 }
 
-if($Field == 'Candidate_Age') {
+if ($Field == 'Candidate_Age') {
     $Field = 'COALESCE(i.Candidate_Age, DATEDIFF(i.Date_taken,c.DoB) / 30)';
 }
-if($FieldX == 'Candidate_Age') {
-    if(isset($InstrumentX)) {
+if ($FieldX == 'Candidate_Age') {
+    if (isset($InstrumentX)) {
         $FieldX = 'COALESCE(i2.Candidate_Age, DATEDIFF(i2.Date_taken,c.DoB) / 30)';
     } else {
         $FieldX = 'COALESCE(i.Candidate_Age, DATEDIFF(i.Date_taken,c.DoB) / 30)';
     }
 }
-$QueryCondition = "$Field IS NOT NULL AND $FieldX IS NOT NULL AND c.Active='Y' and c.Cancelled='N' and s.Active='Y' and s.Cancelled='N'";
-if(isset($_REQUEST['site']) && !empty($_REQUEST['site'])) {
+$QueryCondition = "$Field IS NOT NULL AND $FieldX IS NOT NULL AND c.Active='Y'".
+                  " and c.Cancelled='N' and s.Active='Y' and s.Cancelled='N'";
```

Comment:
Can you just delete this file? The `c.Cancelled` and `s.Cancelled` columns were dropped years ago, which means if anyone tried to use this script it would just die, which means no one is using it.